app/controllers/dashboard_controller.rb

```diff
@@ -8,6 +8,8 @@ def index
     @recent_donations = @donations.recent
     @purchases = current_organization.purchases.includes(:line_items).during(helpers.selected_range)
     @recent_purchases = @purchases.recent
+    @partner_agencies = current_organization.partners.during(helpers.selected_range)
```

Comment:
@jorgedjr21 I'am not entirely sure if this is the correct number we want to show? I think we want show all partner agencies created up till that date. So ```.where('created_at < ?' , Time.current)``` I believe

Reply:
@edwinthinks Well, the issue says that report is by year:
`Number of Partner Agencies in 2020`
So I thought that the report should be filtered by a range

Reply:
@jorgedjr21 ahh yeh I can see how it is confusing. TBH I am not entirely sure either if my assumption is correct. I think it might be worth mentioning to @designbygia. I can ping her to ask.